run/markdown-preview/renderer/main.js

```diff
@@ -0,0 +1,50 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+const express = require('express');
+const MarkdownIt = require('markdown-it');
+
+const app = express();
+app.use(express.text());
+
+app.post('/', (req, res) => {
+  let markdown;
+  if (!req.body) {
+    const msg = 'Markdown data could not be retrieved.';
+    console.log(msg);
+    res.status(400).send(`Error: ${msg}`)
+  } else {
+    markdown = req.body;
+  };
+
+  try {
+    // Get the Markdown text and convert it into HTML using markdown-it.
+    // Markdown-it prohibits some kinds of links making it safe from XSS.
```

Comment:
Yes, the markdown-it docs state it "prohibits some kind of links", including JS. Should this comment explicitly say it protects against JS injection?

Reply:
It seems a bit oddly worded. JS injection is a bit specific, not sure if it matches what markdown-it does. Might be worth reading a bit more into that in the library and about XSS to make sure we understand what protections we are getting... and what we're not.

Reply:
Would it be acceptable to link to the markdown-it security doc in the comments?
i.e. `// Info about  XSS prevention in markdown-it can be found here: https://github.com/markdown-it/markdown-it/blob/master/docs/security.md`

Reply:
yes